### ISSUE TITLE
Using Versionize in a workflow creates an unhandled exception

### ISSUE THREAD

**Issue by maggusw** [saintedlama/versionize#30]
Hi,

```sh
Run versionize --skip-commit --changelog-all -w ./Repo/
  versionize --skip-commit --changelog-all -w ./Repo/
  shell: /bin/bash -e {0}
  env:
    DOTNET_ROOT: /home/runner/.dotnet
Discovered 1 versionable projects
  * /home/runner/work/Repo/Repo.csproj
√ bumping version from 4.0.3 to 4.0.4 in projects
Unhandled exception. System.InvalidOperationException: Remote url https://github.com/organization/Repo is not recognized as valid GitHub HTTPS pattern
   at Versionize.GithubLinkBuilder..ctor(String pushUrl) in C:\p\github\versionize\Versionize\GithubLinkBuilder.cs:line 34
   at Versionize.ChangelogLinkBuilderFactory.CreateFor(Repository repository) in C:\p\github\versionize\Versionize\ChangelogLinkBuilderFactory.cs:line 15
   at Versionize.WorkingCopy.Versionize(Boolean dryrun, Boolean skipDirtyCheck, Boolean skipCommit, String releaseVersion, Boolean ignoreInsignificant, Boolean includeAllCommitsInChangelog) in C:\p\github\versionize\Versionize\WorkingCopy.cs:line 100
   at Versionize.Program.<>c__DisplayClass0_0.<Main>b__0() in C:\p\github\versionize\Versionize\Program.cs:line 36
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass148_0.<OnExecute>b__0(CancellationToken _)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.ExecuteAsync(String[] args, CancellationToken cancellationToken)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at Versionize.Program.Main(String[] args) in C:\p\github\versionize\Versionize\Program.cs:line 50
```

With the https pattern being `"^https://github.com/(?<organization>.*?)/(?<repository>.*?)\\.git$"` but there is no `.git`. Please advise. Workflow running on `ubuntu-latest`.

**Comment by saintedlama** [comment]
Fixed in v1.7.0

**Comment by saintedlama** [comment]
I wonder how this remote URL can be a real github remote URL? Is this just an example?

**Comment by cabauman** [comment]
I ran into this issue locally today, too (due to my remote url not having a `.git` suffix). I'm not sure what determines whether the `.git` suffix is included or not when cloning a repository, but git actions seem to work with or without it. Since I'm working locally at the moment, I was able to do a `git remote set-url` command to make it happy, but might be a good case to handle for a future release.
